clang/test/3C/func_parm_bounds.c

```diff
@@ -0,0 +1,49 @@
+/**
+Test for parameter bounds with parameter with in the declaration.
+Issue: https://github.com/correctcomputation/checkedc-clang/issues/573
+**/
+
+// RUN: rm -rf %t*
+// RUN: 3c -base-dir=%S -alltypes %s -- | FileCheck -match-full-lines -check-prefixes="CHECK_ALL" %s
+// RUN: 3c -base-dir=%S -alltypes %s -- | %clang -c -fcheckedc-extension -x c -o %t1.unusedl -
+// RUN: 3c -base-dir=%S %s -- | %clang -c -fcheckedc-extension -x c -o %t2.unused -
+
+extern _Itype_for_any(T) void *malloc(unsigned long size);
```

Comment:
Since #576, we are using the checked headers instead of hard-coded declarations:
```suggestion
#include <stdlib.h>
```